### PR TITLE
[tests-only] Increase lint timeout to 3 minutes

### PR DESCRIPTION
**Pull request by phil-davis**
This was done in edge branch in PRs #2576 and #2583 

Do it here in master also, to prevent CI fails like:
https://drone.cernbox.cern.ch/cs3org/reva/5884/4/5
```
level=error msg="Timeout exceeded: try increasing it by passing --timeout option"
```

### CODE REVIEW

**Comment by phil-davis** [comment]
@ishank011 @labkode I think that I need a code-owner approval here, and one of you will need to merge this please.

**Review by individual-it — APPROVED** [review]

**Review by ishank011 — APPROVED** [review]

**Pull request merged**
